doc/contributing/tests.md

```diff
@@ -124,6 +124,34 @@ Each acceptance test should be prefixed by `TestAcc_` and should be run using en
 DRIFTCTL_ACC=true go test -run=TestAcc_ ./pkg/resource/aws/aws_instance_test.go
 ```
 
+### Credentials
+
+Acceptance test need credentials to perform real world action on cloud providers:
+- Read/write access are required to perform terraform action.
+- Read only access is required for driftctl execution
+
+Recommended way to run acc tests is to use two distinct credentials,
```

Comment:
credentials: **one** for terraform related actions, and **one** for driftctl scan.